tests/test_integration.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+
+from bbva2pandas import pdf, extractor
+
+
+class TestIntegration(unittest.TestCase):
+    FILEPATH = 'data/bbva_extract.pdf'
+
+    def test_correct_movement_extraction(self):
+        content = pdf.read_pdf(self.FILEPATH)
```

Comment:
Let me know what you think: how about using the [module's API](https://github.com/blalop/bbva2pandas/blob/main/bbva2pandas/__init__.py) in this test? It would require using the `Report` class and checking the dataframe with `to_df`. This way, the integration test does not need to know the inner modules😃